docs/runs.md

```diff
@@ -47,8 +47,10 @@ A `Run` definition supports the following fields:
   - [`metadata`][kubernetes-overview] - Specifies the metadata that uniquely identifies the
     `Run`, such as a `name`.
   - [`spec`][kubernetes-overview] - Specifies the configuration for the `Run`.
-    - [`ref`](#specifying-the-target-custom-task) - Specifies the type and
+    - [`ref`](#specifying-the-target-custom-task-with-ref) - Specifies the type and
       (optionally) name of the custom task type to execute.
+    - [`spec`](#specifying-the-target-custom-task-by-embedding-its-spec) - 
```

Comment:
same here, please replace the reference with `#2-specifying-the-target-custom-task-by-embedding-its-spec`